lib/did-veres-one.js

```diff
@@ -170,6 +170,47 @@ api.generate = util.callbackify(async function(
   return {publicDidDocument, privateDidDocument};
 });
 
+/**
+ * Convert a private key to a public key
+ */
+api.publicDidDocument = util.callbackify(async ({privateDidDocument}) => {
```

Comment:
I seem to remember that you can't use an arrow function with `util.callbackify`.